src/app/home/upcoming-reservations/upcoming-reservations.component.ts

```diff
@@ -16,6 +17,23 @@ import { Reservation } from '../../shared/model/reservation.model';
   styleUrl: './upcoming-reservations.component.scss'
 })
 export class UpcomingReservationsComponent {
-  @Input() reservations: Reservation[] = [];
```

Comment:
Volgens mij is het netter om de data die je in de child components gebruikt in de parent component op te halen en dan via een input door te sturen dus eigenlijk hoe het eerst werd gedaan.